channels/channel_state.go

```diff
@@ -97,7 +96,8 @@ func (c channelState) Voucher() datatransfer.Voucher {
 
 // ReceivedCids returns the cids received so far on this channel
 func (c channelState) ReceivedCids() []cid.Cid {
-	return c.receivedCids
+	receivedCids, _ := c.channelCIDsReader(c.ChannelID())
```

Comment:
Why not return the error from ReceivedCids?

Reply:
just cause it's an API change -- and there's an interface that expects no error. But I will log it.